seahorn/trusty_common/stubs/new_delete_override.cpp

```diff
@@ -0,0 +1,15 @@
+
+#include <cstdlib>
+using namespace std;
+
+void *operator new[](size_t sz) noexcept {
+    if (sz == 0) return nullptr;
+    if (void *ptr = std::malloc(sz))
+        return ptr;
+    return nullptr;
+}
+
+void operator delete(void* ptr) noexcept
```

Comment:
missing `deletep[]`.

I think in C++11 it is sufficient to define `new()` and `delete()` and the array versions are defined automatically based on those